cmd/ci-secret-bootstrap/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/util/homedir"
+
+	"github.com/ghodss/yaml"
+	"github.com/openshift/ci-tools/pkg/util"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift/ci-tools/cmd/ci-secret-bootstrap/bitwarden"
+)
+
+type options struct {
+	dryRun         bool
+	kubeconfigPath string
+	configPath     string
+	bwUser         string
+	bwPassword     string
+
+	kubeconfigs map[string]rest.Config
+	config      *config
+}
+
+func parseOptions() options {
+	var o options
+	fs := flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	fs.BoolVar(&o.dryRun, "dry-run", true, "Whether to actually create the secrets with oc command")
+	fs.StringVar(&o.kubeconfigPath, "kubeconfig", "", "Path to the kubeconfig file to use for CLI requests.")
+	fs.StringVar(&o.configPath, "config", "", "Path to the config file to use for this tool.")
+	fs.StringVar(&o.bwUser, "bw-user", "", "Username to access BitWarden.")
+	fs.StringVar(&o.bwPassword, "bw-password", "", "Password to access BitWarden.")
+	if err := fs.Parse(os.Args[1:]); err != nil {
+		logrus.WithError(err).Errorf("cannot parse args: '%s'", os.Args[1:])
+	}
+	return o
+}
+
+func (o *options) validateOptions() error {
+	for _, arg := range []string{o.bwUser, o.bwPassword, o.configPath} {
+		if arg == "" {
+			return fmt.Errorf("the mandatory arg '%s' is empty", arg)
```

Comment:
This will print `"the mandatory arg '' is empty"`

Reply:
Nit: use `%q`

Reply:
was trying to be start but failed.